profile/CSIP.xml

```diff
@@ -187,7 +187,7 @@
             <requirement ID="CSIP4" REQLEVEL="SHOULD" RELATEDMAT="VocabularyContentInformationTypeSpecification" EXAMPLES="metsRootElementExample1 metsRootElementExample2 metsRootElementExample3">
```

Comment:
I would change the vocabulary name from:

`VocabularyContentInformationTypeSpecification`

to:

`VocabularyContentInformationType`

to match the attribute name.